umap/migrations/0018_datalayer_uuid.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.2.8 on 2024-02-19 17:40
+
+from django.db import migrations, models
+import uuid
+
+
+def gen_uuid(apps, schema_editor):
+    DataLayer = apps.get_model("umap", "DataLayer")
+    for row in DataLayer.objects.all():
```

Comment:
Can't we make Postgres generate the UUID instead ? This python loop can be very busy, in OSM France servers we have more than 2 millions DataLayer.

Reply:
It might be the first step of a 6-steps process(!), see second link. Let's make it work first and then iterate for performances 😅